react/Viewer/TextViewer.jsx

```diff
@@ -61,10 +62,15 @@ export class TextViewer extends React.Component {
   async loadFile() {
     const { url, file } = this.props
     try {
-      const parsedURL = new URL(url)
-      const client = this.props.client.getStackClient()
-      const response = await client.fetch('GET', parsedURL.pathname)
-      const text = await response.text()
+      let resp
+      if (isFileEncrypted(file)) {
+        resp = await fetch(url)
+      } else {
+        const parsedURL = new URL(url)
+        const client = this.props.client.getStackClient()
+        resp = await client.fetch('GET', parsedURL.pathname)
+      }
+      const text = await resp.text()
```

Comment:
let's test this feature 🙏

Reply:
Sure [db7da7d](https://github.com/cozy/cozy-ui/pull/2112/commits/db7da7d8e7fc1ac7e183ee4c3759d66cc23b8042)